app/views/shelters/show.html.erb

```diff
@@ -9,10 +9,11 @@
   <div class="shelter_options"><%= link_to "See Our Pets", "/shelters/#{@shelter.id}/pets" %></div>
 
   <section class="review-section">
+    <%= link_to ("New Review"), "/shelters/#{@shelter.id}/reviews/new" %>
     <% @shelter.reviews.each do |review| %>
       <%= image_tag ( review.review_image ) %>
       <%= review.title %>
-      <%= review.rating %>
+      Rating: <%= review.rating %>
```

Comment:
We need to load a DB and see what this formatting looks like  with the rating just hanging out.

Reply:
unless it has a label it will just be a line with

3

and no reference to what it is, or why there is a 3 there.

Reply:
currently all on one line. for formatting we may want to change to a list or throw a <br> after each review item